crates/sargon/src/core/utils/constants.rs

```diff
@@ -19,3 +19,10 @@ pub const GATEWAY_CHUNK_NON_FUNGIBLES: u64 = 100;
 
 // Max amount of addresses to be queried in one request to `/state/entity/details/`.
 pub const GATEWAY_ENTITY_DETAILS_CHUNK_ADDRESSES: u64 = 20;
+
+// Maximum number of characters allowed for an entity name
+pub const ENTITY_NAME_MAX_LENGTH: u64 = 30;
```

Comment:
Ah we have this already, `DisplayName::MAX_LEN` so please delete this and use `DisplayName::MAX_LEN` inside `sargon-uniffi`